data/card_catalogue.json

```diff
@@ -628,87 +670,87 @@
         {
           "title": "Whip",
           "upgrade": "Pitch",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "Throw an object up to 10m",
+            "rules": "Throw an object up to 10 squares",
             "flavour": "\"Real Good\" -Divo",
             "image": "assets/core/card/illustration/Whip.png"
           }
         },
         {
           "title": "Pitch",
           "upgrade": "Hurl",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "Throw an object up to 50m",
+            "rules": "Throw an object up to 50 squares",
             "flavour": "\"I will not rest until every window is smashed. They are an insult to me and my kind\" -Little Jimmy, neighborhood rascal",
             "image": "assets/core/card/illustration/Pitch.png"
           }
         },
         {
           "title": "Hurl",
           "upgrade": "Launch",
-          "tier": "green",
+          "tier": "red",
           "details": {
-            "rules": "Throw an object up to 300m",
+            "rules": "Throw an object as far as you want.",
```

Comment:
1. There wasn't enough differential between red and gold
2. It didn't have plot repercussions which we'd like for gold cards